discord/ext/commands/errors.py

```diff
@@ -26,7 +26,7 @@
 
 from typing import Optional, Any, TYPE_CHECKING, List, Callable, Type, Tuple, Union
 
-from discord.errors import ClientException, DiscordException
+from discord.errors import *
```

Comment:
Is a wildcard import needed here?

Reply:
I don't know if it's been removed, but originally you can access all the `discord.errors` from `commands.errors`.
I can revert this if needed